tests/common.py

```diff
@@ -1,7 +1,7 @@
 import os
 
 MASTER_KEY = 'masterKey'
-BASE_URL = os.getenv('MEILISEARCH_HOST') if os.getenv('MEILISEARCH_HOST') else 'http://127.0.0.1:7700'
+BASE_URL = os.getenv('MEILISEARCH_URL') if os.getenv('MEILISEARCH_URL) else 'http://127.0.0.1:7700'
```

Comment:
This should fix the failing tests.

```suggestion
BASE_URL = os.getenv('MEILISEARCH_URL') if os.getenv('MEILISEARCH_URL') else 'http://127.0.0.1:7700'
```

Reply:
Hi @sanders41,
Thank you very much for your suggestions which make the code cleaner however another PR has been opened on this issue without any error so I will merge the other one instead

Reply:
Apologies, I couldn't do the required changes. I went to office, and in office personal laptops aren't allowed. 
Seems like, my other PR in php will also show error _sigh_

Reply:
No worries. Maybe next time will be the good one!